universal-application-tool-0.0.1/test/support/TestQuestionBankTest.java

```diff
@@ -29,4 +34,10 @@ public void withoutDatabase_setsId() {
 
     assertThat(question.id).isEqualTo(2L);
   }
+
+  @Test
+  @Parameters(source = QuestionType.class)
+  public void getSampleQuestionsForAllTypesTest(QuestionType questionType) {
```

Comment:
(blocking) this should be named `getSampleQuestionForAllTypes_hasAllTypes`.

Reply:
Resolved